src/editor/JSTTextView.m

```diff
@@ -51,18 +54,64 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
     
     self = [super initWithCoder:aDecoder];
 	if (self != nil) {
-        // what's the right way to do this?
-        [self performSelector:@selector(setupLineViewAndStuff) withObject:nil afterDelay:0];
+        [self initThemes];
+        [self setupLineViewAndStuff];
         [self setSmartInsertDeleteEnabled:NO];
         [self setAutomaticQuoteSubstitutionEnabled:NO];
     }
     
     return self;
 }
 
+- (void)initThemes {
+    self.darkCodeHighlightingColors = @{
```

Comment:
This would be neater as a dictionary of dictionaries, keyed by the theme:

```
@interface JSTTextView ()
…
@property (copy) NSDictionary *themeHighlightingColors;
@end

@implementation JSTTextView
…
- (void)initThemes {
     self.themeHighlightingColors[@(JSTTextViewThemeLight)] = @{…};
     self.themeHighlightingColors[@(JSTTextViewThemeDark)] = @{…};
}
…
@end